src/Components/Home/HomeEditForm.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from 'react';
+//Forms
+import { useFormik } from 'formik';
+import * as Yup from 'yup';
+//Styles
+import Swal from 'sweetalert2';
+import {
+  Button,
+  Typography,
+  Container,
+  TextField,
+  Paper,
+  Stack,
+} from '@mui/material';
+import s from '../../Styles/HomeEditForm.module.css';
+//Components
+import SlidesForm from '../Slides/SlidesForm';
+
+const validationSchema = Yup.object({
+  welcomeText: Yup.string()
+    .min(20, 'El texto de bienvenida debe ser de al menos 20 caracteres')
+    .required('Este campo es obligatorio'),
+});
+
+export default function HomeFormEdit() {
+  const [welcomeText, setWelcomeText] = useState('');
+
+  const formik = useFormik({
+    initialValues: {
+      welcomeText: '',
+    },
+    validationSchema,
+    onSubmit: (values, { setSubmitting }) => {
+      Swal.fire({
+        text: 'El texto de bienvenida se actualizó correctamente!',
+        icon: 'success',
+        confirmButtonText: 'Hecho',
+      });
+      setWelcomeText(values.welcomeText);
+      setSubmitting(false);
+    },
+  });
+
+  return (
+    <Container maxWidth="md" sx={{ marginBlock: '4rem' }}>
+      <Typography
+        color="initial"
+        sx={{ fontFamily: 'Product Sans' }}
+        variant="h3">
+        Editar página de inicio
+      </Typography>
+      <Paper
+        elevation={3}
+        sx={{
+          padding: '2rem',
+          marginBlock: '4rem',
+        }}>
+        <form
+          sx={{
+            display: 'flex',
+            justifyContent: 'center',
+            gap: '4rem',
+            flexDirection: 'column',
+          }}
```

Comment:
the styles should be in style file on Styles. 
```
{
            display: 'flex',
            justifyContent: 'center',
            gap: '4rem',
            flexDirection: 'column',
          }
```
its an object so you can create the object on the styles file that correspond and then you can call it here

Reply:
Idk if I solved this properly. I 've created 3 files in Styles folder for the same component. Is it ok?